service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterStatementTemplateParameters.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class EncounterStatementTemplateParameters {
+    private String encounterStatementId;
+    private String status;
+    private String effectiveTime;
```

Comment:
From the Mapping spreadsheet we need to populate availability time with period start